Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -187,7 +188,9 @@ extension WebViewController: WKNavigationDelegate {
         let allowDecision = WKNavigationActionPolicy(rawValue: WKNavigationActionPolicy.allow.rawValue + 2) ?? .allow
 
         let decision: WKNavigationActionPolicy = RequestHandler().handle(request: navigationAction.request, alertCallback: present) ? allowDecision : .cancel
-
+        if navigationAction.navigationType == .linkActivated && browserView.url != navigationAction.request.url {
```

Comment:
Do we want to track every time a user taps a link? Or changes websites? cc: @bbinto

Reply:
Taps a link.

Reply:
FYI the point here is to send events as the user takes action so we know the the user is active - so any link will do.

Reply:
👍